src/component/ComposedTwoYAxisChart.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+	Brush,
+	Line,
+	XAxis,
+	YAxis,
+	Tooltip,
+	Legend,
+	ComposedChart
+} from "recharts";
+
+function getRandomArbitrary(min: number, max: number) {
+	return Math.ceil(Math.random() * (max - min) + min);
+}
+
+var data: any[] = [];
+for (let i = 0; i < 2000; i++) {
+	let d = {
+		bitrate: getRandomArbitrary(3800, 4200),
+		framerate: getRandomArbitrary(59, 61)
+	};
+	data.push(d);
+}
+data[900]["bitrate"] = 2300;
+data[952]["bitrate"] = 2900;
+data[1000]["bitrate"] = 4500;
+data[1001]["bitrate"] = 4500;
+data[1022]["bitrate"] = 4300;
+data[1030]["bitrate"] = 4500;
+
+export default function App() {
```

Comment:
default function exports don't need to be named. `export default function() {` is sufficient.